utils/cache/backend_redis.c

```diff
@@ -328,3 +356,16 @@ status_t cache_list_pop(const char* const key, char* res) {
   }
   return redis_list_pop(CONN(cache)->rc, key, res);
 }
+
+long int cache_occupied_space() {
+  if (!cache_state) {
```

Comment:
Can we avoid using global variables such as `cache_state`? We might suffer from potential data racing.

Reply:
I will add this into the aim of #556 .
The implementation in this PR disable redis service iff buffered transactions exceeds the maximum capacity. This scenario is a relative extreme situation.